### PR TITLE
Update config.php

### PR DESCRIPTION
**Pull request by jospoortvliet**
How about we get 12.0.3 to 50% of our users via the updater? No big disasters have shown up yet ;-)

### CODE REVIEW

**Comment by jospoortvliet** [comment]
How do we get this deployed, @nickvergessen @MorrisJobke you know?

**Comment by MorrisJobke** [comment]
> How do we get this deployed, @nickvergessen @MorrisJobke you know? Or is this automatic?

Somebody™ needs to trigger it. Let me do it.

**Comment by jospoortvliet** [comment]
thanks, @MorrisJobke you rock!

File: config/config.php [hunk 1]
```diff
@@ -194,20 +194,20 @@
...
-			],	*/	
-			'100' => [
+				'signature' => 'YDatIPqOzjHGdV4sCKAqOHPZORN4k7FswMpfTl8S7JlyRW0aLOxA8KUyBmxmfgVY
+lMujKm2gtCRHGEs7QrG+J5NeJ5g9PS1vWsrU1mCpGgLz1ifjF5VfpSuSTWK3WxpS
+b0aRy+YfJNrfZjYHGFiL/KDlr0+Eu+MWSKHNbM5YZBZcYCbgSQZH4ry2AgMPAqRq
+qaZ9jV3c0nObBq4Ha2qDAmBhcSnVMVmoXrrumgVyzSC+F7CLf7X34TjEPw/hFzr2
+N4FMRE9PHpYqRqoRF+OKNYvcs3MVAiv2QTSjJd2wtiutfaDO8zJUsd/CoTbPr750
+0ee22KtYHJFmotmZ9EjwIg==',
+			],		
+			'50' => [
```
Comment (MorrisJobke) [inline config/config.php]: PHP says no: duplicate key. Let me fix that.

Comment (MorrisJobke) [reply config/config.php]: c471f61

**Comment by MorrisJobke** [comment]
Tested, works and deployed.

**Pull request merged**